Frontend/CO.CDP.OrganisationApp/Pages/Organisation/Search.cshtml

```diff
@@ -0,0 +1,132 @@
+@page "/organisation/search/buyer"
```

Comment:
URL has changed?
I've linked to what you told me previously. If you want it to be specific to buyers, I think organisation/buyer/search is a better option as it matches the existing convention